pages/linux/kscreen-console.md

```diff
@@ -0,0 +1,32 @@
+# kscreen-console
+
+> KScreen Console is a CLI tool to query KScreen's status.
+> More information: <https://manned.org/kscreen-console>.
+
+- Show all outputs and configuration files to attach to a bug report:
+
+`kscreen-console bug`
+
+- Show paths to KScreen configuration files:
+
+`kscreen-console config`
+
+- Show display output information:
+
+`kscreen-console outputs`
+
+- Monitor for changes:
+
+`kscreen-console monitor`
+
+- Show the current KScreen configuration in JSON:
```

Comment:
```suggestion
- Show the current KScreen configuration in JSON format:
```

Reply:
Shouldn't that `a JSON format`

Reply:
I've noticed a lot of the repository uses "in JSON format" like this as well.
I can't say I'm the fondest of it, but it would certainly be more consistent with the rest of the repository.

Just want to make room for a discussion now, we sure the word "format" should even be there? 
Native English speakers wouldn't say "in JSON format", we just say "in JSON", same with HTML, XML, etc.

Suppose the following:
* The output is in JavaScript Object Notation format.
* The output is in Hypertext Markup Language format.
* The output is in Extensible Markup Language format.

In all 3 of these cases, the word format is redundant and rarely said.

@CleanMachine1 Also I don't think it'd be "a JSON format", since that would suggest there are multiple JSON formats. The word "a" would be valid for "a JSON file" or "a JSON schema". I don't think it's valid for "a JSON format".

Reply:
If we're going to be strictly grammatically correct here, then I'd suggest this:

```suggestion
- Show the current KScreen configuration as JSON:
```

...or

```suggestion
- Show the current KScreen configuration formatted as JSON:
```

.....but I see `in SOMETHING format` often enough and it's not enough of a problem for me to point out on it's own. It falls into the bin of "things that I'll suggest an improvement to if I find something else to suggest a change for, but if I don't I'll just leave it" for me.

Reply:
I concede to the "as JSON" suggestion, I think that's better.
The other suggestion is fair too, but I think keeping things concise as long as it doesn't lead to ambiguity.

Sorry if I'm caring too much about grammar. ^-^'

Reply:
>Sorry if I'm caring too much about grammar. ^-^'

You're not, I'm also bothered by this.  I like @sbrl's first suggestion because it's shorter, but both are more fluent.

I think that `in JSON format` is not really acceptable — it's pretty bad grammar and not easy to read.

Let's go with the first suggestion.